compiler/rustc_feature/src/unstable.rs

```diff
@@ -54,6 +56,10 @@ impl Features {
         self.enabled_features.insert(name);
     }
 
+    /// Returns a list of triples with:
+    /// - feature gate name
+    /// - the span of the `#[feature]` attribute
+    /// - (for already stable features) the version since which it is stable
     pub fn enabled_lang_features(&self) -> &Vec<(Symbol, Span, Option<Symbol>)> {
```

Comment:
Discussion: not for this PR, but like

```rs
pub struct LangFeature {
    name: Symbol,
    attr_sp: Span,
    stable_since: Option<Symbol>,
}

pub fn enabled_lang_features(&self) -> &Vec<LangFeature> { ... }
```

Reply:
The type should probably be called `EnabledLangFeature`, but other than that -- yeah makes sense.